### DISCUSSION TITLE
Is beta.5 stable now?

### DISCUSSION THREAD

**Discussion by pranay01** [facebook/docusaurus#5448] [General] [answered]
I see many interesting SEO features shipped in beta5. Is it stable to migrate to now?

**Comment by slorber** [comment, marked as answer]
Yes it should be fine. 
I'm not even sure to understand what you mean. 
What makes you think beta 5 is less stable than beta 4? Maybe it is, but we can't know if you don't upgrade and tell us 😅

**Reply by pranay01** [reply]
ha ha, ok :)
